build.sbt

```diff
@@ -201,6 +201,11 @@ lazy val chisel = (project in file(".")).
     )
   )
 
+lazy val noPluginTests = (project in file ("no-plugin-tests")).
```

Comment:
why was it necessary to add this `noPluginTests` as part of this PR?

Reply:
I noticed that all of the tests passed, but I had a hunch that things wouldn't work without it. Hunch turned out to be correct so I wanted to make sure the no plugin behavior was supported.